internal/admin/settings_apply.go

```
@@ -0,0 +1,166 @@
+// auto-optimize toggle established: when a flip cannot take effect,
+// saying so with the reason is what gives the operator something to act
+// on, where a silent success would have them flip the switch, watch
+// nothing happen, and have nothing to go on.
+func (a applyReport) restartBecause(field, reason string) {
+	a.set(field, applyRestart, reason)
+}
+
+// changed records live-or-restart in one call, for the many sites whose
+// only variable is whether the consumer reads the holder or captured a
+// boot snapshot.
+func (a applyReport) changed(field string, hot bool) {
```

golangci-lint (2.12.2)

[error] func applyReport.changed is unused

(unused)